packages/mookme/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@escape.tech/mookme",
-  "version": "2.2.0",
+  "version": "2.4.0",
```

Comment:
The beta branch might not have hotfixes of the main branch, but I'll handle fixing versions, should be part of a separate merge request :)

Reply:
Yeah I'm not sure where that came from

Reply:
It seems to have picked up a couple of old commits. I can start over, but they're hanging out here right now.

Reply:
versions are released by hand, so I can fix versions later